src/main/update.ts

```diff
@@ -190,35 +190,36 @@ export async function update(update: Update, listeners: IUpdateOptions) {
   console.log("Finished to download:", dlPath);
 
   const extractPath =
-    process.platform == "darwin" // .app으로 실행한 건지 npm run dev로 실행한 건지도 확인해야 함
+    process.platform == "darwin" // we should check whether it was executed from .app or from npm run dev.
       ? path.dirname(path.dirname(path.dirname(path.dirname(app.getAppPath()))))
       : path.dirname(path.dirname(app.getAppPath()));
   console.log("The 9C app installation path:", extractPath);
 
   const appDirName = app.getAppPath();
-  // FIXME: "config.json" 이거 하드코딩하지 말아야 함
+  // FIXME: We shouldn't hardcode "config.json"
```

Comment:
A GFI candidate, although unsure how this can be done. Maybe we should ask the author.